README.md

```diff
@@ -9,4 +9,4 @@
 - [ ] Day 7
 - [ ] Day 8
 - [ ] Day 9
-- [ ] Day 10
\ No newline at end of file
+- [ ] Day 10
```

Comment:
Só uma observação: quando for dar um squash, seria interessante esse arquivo aqui ir em um commit separado, porque ele tá em outro contexto.